tensorboard/webapp/notification_center/_redux/BUILD

```diff
@@ -0,0 +1,45 @@
+load("//tensorboard/defs:defs.bzl", "tf_ts_library")
+
+package(default_visibility = ["//tensorboard/webapp/notification_center:__subpackages__"])
+
+licenses(["notice"])
+
+tf_ts_library(
+    name = "_redux",
+    srcs = [
+        "notification_center_actions.ts",
```

Comment:
even in this folder structure, I think actions should be separated out so the view that wants to dispatch an action does not have to import all of the redux module. 

What do others think?

Reply:
Haven't really got into the details of actions so I am not sure about this. But I think we can also come back to address them. For now I separate the actions from the rest of files.

Reply:
iiuc, view modules don't need to import a "redux module" at all?  If a view imports actions directly from the actions file, I don't think it matters too much whether the view's build target has a dep on ":_redux" vs ":actions".

I'm also not super keen on the re-exported private view module approach, which requires an extra boilerplate file compared to the previous `filegroup` approach.  Since there is a plan to discuss folder structure details soon, I don't want to block this PR if separating seems reasonable.